client/src/Pages/Profile.tsx

```diff
@@ -1,62 +1,92 @@
-import React, { useEffect, useState } from 'react';
-import { WorkCard, UserInfoCard } from '../Components';
+import React, { useEffect, useState, useContext } from 'react';
+import { useParams } from 'react-router-dom';
 import {
-  Works, Request, AllWorks, OnWork, request,
+  UserInfoCard, SpinierComponent, WorkList, Reviews, ErrorComponent,
+} from '../Components';
+import {
+  Works, User, TopTenReviews, ProfileDataProps, WorksProps,
 } from '../utils';
+import { UserContext } from '../Context/LoggedUserContext';
+import getUserProfileData from '../Controllers/getProfileData';
+import getWorksData from '../Controllers/getWorks';
```

Comment:
make an index file in the controller's folder and import getWorksData, getProfileData from it directly